<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MelodicSetupForm</class>
 <widget class="QWidget" name="MelodicSetupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>455</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>455</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_38">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Obs: C3 = Dó central</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>343</height>
    </rect>
   </property>
   <property name="title">
    <string>Notas Permitidas</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>C</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>C#/Db</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>D</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>D#/Eb</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>E</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>F</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>F#/Gb</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>G</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>G#/Ab</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>A</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>A#/Bb</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>B</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QSpinBox" name="spinBoxC"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxDb"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxD"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxEb"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxE"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxF"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxGb"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxG"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxAb"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxA"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxBb"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxB"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>215</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelLimiteSuperior">
        <property name="text">
         <string>Limite Superior</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelLimiteInferior">
        <property name="text">
         <string>Limite Inferior</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEditLimiteSuperior"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditLimiteInferior"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>221</width>
     <height>343</height>
    </rect>
   </property>
   <property name="title">
    <string>Intervalos Permitidos</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QSpinBox" name="spinBoxUnissono"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox2m"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox2M"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox3m"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox3M"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox4J"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox4aum"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox5J"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox6m"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox6M"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox7m"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox7M"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox8J"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Uníssono</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>2ªm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>2ªM</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>3ªm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>3ªM</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>4ªJ</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>4ªaum</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>5ªJ</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>6ªm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>6ªM</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>7ªm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>7ªM</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>8ªJ</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>9ªm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>9ªM</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>10ªm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>10ªM</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>11ªJ</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>11ªaum</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>12ªJ</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>13ªm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_34">
          <property name="text">
           <string>13ªM</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_35">
          <property name="text">
           <string>14ªm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_36">
          <property name="text">
           <string>14ªM</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_37">
          <property name="text">
           <string>15ªJ</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QSpinBox" name="spinBox9m"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox9M"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox10m"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox10M"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox11J"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox11aum"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox12J"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox13m"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox13M"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox14m"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox14M"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox15J"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
